### ISSUE TITLE
How to create presigned POST

### ISSUE THREAD

**Issue by hasufell** [brendanhay/amazonka#533]
https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#createPresignedPost-property

Can't find anything like this and amazonka documentation is honestly quite lacking.

**Comment by endgame** [comment]
I have a hunch that the primitives exist in `Network.AWS.Presign`, but looking at it closely probably requires picking over an AWS SDK, and that can happen after 2.0.

**Comment by endgame** [comment]
What presigning are you looking to do? A basic "here, POST your data to this one URL" or are you trying to integrate complex conditions into your request?

**Comment by hasufell** [comment]
I have no idea. This ticket is 6 years old and the company I worked for at the time is broke.

**Comment by tchoutri** [comment]
@endgame I'm writing a file storage service and this is exactly my usecase: "Here, POST your data to this one URL".

**Comment by tchoutri** [comment]
Well, I guess I was wrong to look in `-s3`, apparently it exists in https://hackage.haskell.org/package/amazonka-2.0/docs/Amazonka-Presign.html

**Comment by chreekat** [comment]
What remains of this issue to satisfy the 2.1 milestone? @tchoutri did you make it work? Is it just a documentation effort, or is the functionality still missing? If the latter, is it really a blocker for 2.1?

**Comment by tchoutri** [comment]
@chreekat yes as my comment indicated, the functionality was simply elsewhere. I just had to adjust my mental model because S3 is probably not the only AWS service that wants presigned URLs. :) All good on my end!

**Comment by chreekat** [comment]
Great. :) My reason for asking is that Stackage depends on unreleased patches and I was wondering how/if we could reduce the scope of 2.1. Seems this issue is now a documentation issue *at best*, so perhaps it could be moved or closed....
